src/blocks/container/backgroundCss.js

```diff
@@ -130,6 +181,24 @@ const backgroundCss = ( attributes, deviceType, clientId, pseudoElementOverlay =
 		xPositionType: xPositionTypeTablet ? xPositionTypeTablet : backgroundAttributesDesktop.xPositionType,
 		yPosition: ! isNaN( yPositionTablet ) ? yPositionTablet : backgroundAttributesDesktop.yPosition,
 		yPositionType: yPositionTypeTablet ? yPositionTypeTablet : backgroundAttributesDesktop.yPositionType,
+
+		backgroundOverlayImage: backgroundOverlayImageTablet || backgroundOverlayImageDesktop,
+		backgroundOverlayRepeat: backgroundRepeatOverlayTablet || backgroundRepeatOverlayDesktop,
+		backgroundOverlayPosition: backgroundPositionOverlayTablet || backgroundPositionOverlayDesktop,
+		backgroundOverlaySize: backgroundSizeOverlayTablet || backgroundSizeOverlayDesktop,
+		backgroundOverlayAttachment: backgroundAttachmentOverlayTablet || backgroundAttachmentOverlayDesktop,
+		backgroundCustomOverlaySize: backgroundCustomSizeOverlayTablet || backgroundCustomSizeOverlayDesktop,
+		backgroundCustomOverlaySizeType,
+		customOverlayPosition,
+		xPositionOverlay: ! isNaN( xPositionOverlayTablet ) ? xPositionOverlayTablet : backgroundAttributesDesktop.xPositionOverlay,
```

Comment:
Will `! isNaN` work as intended here?

```suggestion
		xPositionOverlay: ( 'number' === typeof xPositionOverlayTablet ) ? xPositionOverlayTablet : backgroundAttributesDesktop.xPositionOverlay,
```